File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        
      HashMap<Character,Integer>m = new HashMap<>();
        
        for(int i=0 ;i<sentence.length() ;i++)
        {
            if(!m.containsKey(sentence.charAt(i)))
            {
                m.put(sentence.charAt(i),1);
            }
            
            else
            
            m.put(sentence.charAt(i),m.get(sentence.charAt(i))+1);
        }
        
        for(char c ='a'; c<='z' ; c++)
        {
            if(!m.containsKey(c))
                return false;
        }
        
        return true;
    }
}

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        
        if(sentence.length()<26)
            return false;
        
        else
        {
            unordered_map<char,int>m;
            
            for(int i=0;i<sentence.length();i++)
            {
                m[sentence[i]]++;
            }
            
            for(char c='a' ; c<='z';c++)
            {
                if(m.find(c)==m.end())
                    return false;
            }
            
        }
        
        return true;
    }
};